### ISSUE TITLE
pre-trained fcn32s_model_iter_XXX.h5 seems to be unuseful!

### ISSUE THREAD

**Issue by dxiaosa** [wkentaro/fcn#81]
When I use the fcn32s_model_iter_XXX.h5 model as the pre-trained model to train fcn16s, the procedure can run, but it decreased the accuracy and make the segmentation worse in the same conditions.

**Comment by yasuru** [comment]
I have also met this problem. However, this is still not resolved. I doubt the following points； 
・code of "crop"　(How is the offset value determined?)
・The number of iterations is not enough (but, the profile of loss seems to be saturated...)
Please let me know when you find out.

**Comment by wkentaro** [comment]
In my case:

I used FCN32s-apc2016v2 (val/iu ~ 0.83, iteration ~ 100000) to train FCN16s-apc2016v2, and got val/iu ~ 0.862 with iteration < 89000.

**Comment by wkentaro** [comment]
I will try training FCN16s-pascal, and let you know the result.

**Comment by Hjy20255** [comment]
@wkentaro did you train FCN16s-pascal model ? 
I have also met this problem. However, this is still not resolved.

**Comment by wkentaro** [comment]
@Hjy20255 Sorry, actually no. I'm migrating to PyTorch, and do it with Chainer if I have time.

**Comment by wkentaro** [comment]
Now it works: https://github.com/wkentaro/fcn/issues/89#issuecomment-306586111
